### PR TITLE
WAS-373 business address page

### PR DESCRIPTION
**Pull request by rickclare**
Note:  This merge goes side-to-side with this PR: https://github.com/EnvironmentAgency/digital-services-core/pull/27 

These two PRs need to be merged at the same time, and then I need to remember to change this line to point to the "develop" branch: https://github.com/EnvironmentAgency/waste-exemptions/blob/feature/was-373-business-address-page/Gemfile#L28

### CODE REVIEW

**Comment by rickclare** [comment]
@autotelik Note: I've yet to merge in your recent changes from your recently merge "Farmer Page" feature.

**Comment by autotelik** [comment]
code looks fine to me. @Cruikshanks  when you are happy with PR,  I'm happy to do the manual merge - as pretty sure the conflicts will be FactoryGirl related - farmer stuff now in develop

File: app/controllers/concerns/waste_exemptions/after_state_updated.rb [hunk 1]
```diff
@@ -22,5 +22,12 @@ def after_not_known(enrollment, params)
       end
     end
 
+    def after_organisation_address(enrollment, params)
+      enrollment.organisation.contact.addresses.try(:first).try(:update!, address_type: :primary)
```
Comment (Cruikshanks) [inline app/controllers/concerns/waste_exemptions/after_state_updated.rb]: Do we expect there to be at least 1 address at this point? The reason i ask is if we do, is ```try()``` hiding an exception from us which should actually be raised. Else should we be doing something else if try we're to return ```nil```?

Comment (autotelik) [inline app/controllers/concerns/waste_exemptions/after_state_updated.rb]: or perhaps a sign that there is a method missing on Organisation .. something like ... set_contact_address_type( address_type)  

 or   a missing assocaition  something like  ... has_one primary_contact_address

Comment (rickclare) [inline app/controllers/concerns/waste_exemptions/after_state_updated.rb]: There might not be any address, due to validation errors - I think that's why I coded it this way.

Comment (rickclare) [inline app/controllers/concerns/waste_exemptions/after_state_updated.rb]: Probably just me coding over-defensively.  I will remove the `try` wrappers and resort to regular method calls.   We can then deal with exceptions more smartly, if we find they do occur.

**Comment by rickclare** [comment]
@Cruikshanks Amended implemented as per your PR comments.  Over to you for final review and merge.

**Comment by Cruikshanks** [comment]
This is good to go, though @autotelik has kindly taken on the task of resolving the merge commits. Might just want to run it past @rickclare when done. Also don't overlook Rick's comment at the start of this PR > and then I need to remember to change this line to point to the "develop" branch: https://github.com/EnvironmentAgency/waste-exemptions/blob/feature/was-373-business-address-page/Gemfile#L28

**Pull request merged**